applet/src/keyb.c

```diff
@@ -119,20 +128,17 @@ void handle_hotkey( int keycode )
             syslog_dump_dmesg();
             break ;
         case 7 :
```

Comment:
look. if you dont present the code, it cannot be evaluated. dont whine about it.